<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTimePage</class>
 <widget class="QWidget" name="SetTimePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:0px;
font:25px &quot;SimHei&quot;;
color:black;
}

QPushButton
{
font:25px, &quot;SimHei&quot;;
}

/*QPushButton:disabled
{
outline:none;
border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(180, 180, 180);
}*/</string>
  </property>
  <widget class="QPushButton" name="btnYear">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMonth">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDay">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHour">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinute">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSecond">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
font:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
